### DISCUSSION TITLE
[TRILHA EXISTENTE] Python - Adição de Tópicos (SO, Arquivos, Fontes Externas).

### DISCUSSION THREAD

**Discussion by phsmith** [codaqui/institucional#73] [Trilhas de Ensino]
O que acham de adicionarmos os seguintes tópicos na trilha de python:

- Interagindo com o sistema operacional
- Manipulando arquivos
- Consumindo dados de fontes externas

**Comment by endersonmenezes** [comment]
@phsmith perfeito! Pode começar montar os textos para nós.

Issue: https://github.com/codaqui/institucional/issues/74

**Reply by phsmith** [reply]
Boa! Valeu @endersonmenezes.

**Comment by endersonmenezes** [comment]
PR: #78
